README.md

```diff
@@ -118,3 +118,32 @@ const App = () => {
    - spotId
    - postId
    - (optional) className
+
+## Two Token Authentication
```

Comment:
Nice! I believe you should consult with Kevin to document this also on our documentation hub.